client/layout/guided-tours/tours/activity-log-tour.js

```diff
@@ -64,7 +65,7 @@ export const ActivityLogTour = makeTour(
 		<Step name="expand" arrow="top-left" target=".has-expanded-summary" placement="below">
 			<p>
 				{ translate(
-					'You can also click on each daily block to show more details. ' +
+					'You can click on each block to show more details. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'You can also click on each daily block to show more details. When expanded, it will display every event that happened on that day.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).